plugins/legacy-plugin-chart-histogram/src/WithLegend.jsx

```diff
@@ -117,13 +117,18 @@ class WithLegend extends React.Component {
       flexDirection: legendDirection,
       justifyContent: this.getLegendJustifyContent(),
     };
-
+    const legendContainerStyle = {
+      flexWrap: 'wrap',
+      display: 'flex',
+      flexDirection: 'row',
```

Comment:
I wonder if we should do this, just in case (it appears the controls don't support customizing these yet)
```suggestion
      flexDirection: legendDirection,
```